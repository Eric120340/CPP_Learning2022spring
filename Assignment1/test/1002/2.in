4 7
34667 54970 85137 31392 72268 16697 79196 
71237 97656 38591 2088 86075 65671 18075 
43050 75180 27904 9844 39181 17918 81654 
98856 88703 91207 85138 79836 89611 15921 
4 7
56453 61285 40937 91120 32606 26073 38863 
4873 42769 18058 76110 40425 73000 78197 
42851 55023 12623 2252 46554 56879 28447 
2086 74796 10101 941 63498 1307 2430 
