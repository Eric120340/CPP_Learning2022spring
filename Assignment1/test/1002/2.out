91120 116255 126074 122512 104874 42770 118059 
76110 140425 56649 78198 126500 138671 96272 
85901 130203 40527 12096 85735 74797 110101 
100942 163499 101308 86079 143334 90918 18351 
